### PR TITLE
jinx: adds FFNG's mirror and gtk3 repo seeds

### PR DESCRIPTION
**Pull request by sandersch**
Updates the default seeded repos. Add's @FarFigNewGut's repo for everyone and the gtk3 script updates repo if the Gtk3 constant is installed.

### CODE REVIEW

File: scripts/jinx.lic [hunk 1]
```diff
@@ -185,6 +185,16 @@ module Jinx
...
       url: %[https://extras.repo.elanthia.online]
     }
+
+    FFNGLichRepoArchiveMirror = {
+      name: "mirror",
+      url: %[https://ffnglichrepoarchive.netlify.app]
+    }
+
+    Gtk3ScriptUpdates = {
+      name: "mirror",
```
Comment (ondreian) [inline scripts/jinx.lic:195]: shouldn't this be gtk3?

Comment (sandersch) [reply scripts/jinx.lic]: whoops. good catch, fixed.

**Review by ondreian — APPROVED** [review]
lgtm, thanks

**Comment by FarFigNewGut** [comment]
🎉

**Pull request merged**